docs/excel/excel-add-ins-workbooks.md

```diff
@@ -132,17 +132,64 @@ A workbook's settings are similar to the collection of custom properties. The di
 
 ```js
 Excel.run(function (context) {
-	var settings = context.workbook.settings;
-	settings.add("NeedsReview", true);
-	var needsReview = settings.getItem("NeedsReview");
-	needsReview.load("value");
-
-	return context.sync().then(function() {
-		console.log("Workbook needs review : " + needsReview.value);
-	});
+    var settings = context.workbook.settings;
+    settings.add("NeedsReview", true);
+    var needsReview = settings.getItem("NeedsReview");
+    needsReview.load("value");
+
+    return context.sync().then(function() {
+        console.log("Workbook needs review : " + needsReview.value);
+    });
 }).catch(errorHandlerFunction);
 ```
 
+## Add custom XML data to the workbook
+
+Excel's Open XML **.xlsx** file format lets your add-in embed custom XML data in the workbook. This data persists with the workbook, independent of the add-in.
+
+A workbook contains a [CustomXmlPartCollection](/javascript/api/excel/excel.customxmlpartcollection), which is a list of [CustomXmlParts](/javascript/api/excel/excel.customxmlpart). These give access to the XML strings and a corresponding unique ID. By storing these IDs as settings, your add-in can maintain the keys to its XML parts between sessions.
```

Comment:
shouldn't the link be to **excel.customxmlparts** instead of excel.customxmlpart?